smart_contract/src/lib.rs

```diff
@@ -489,14 +499,12 @@ mod tests {
     const ISI_RESULT: Result<(), ValidationFail> = Ok(());
 
     fn get_test_instruction() -> InstructionBox {
-        let new_asset_id = "tulip##alice@wonderland".parse().unwrap();
-        let register_isi = Register::asset(Asset::new(new_asset_id, 1_u32));
-
-        register_isi.into()
+        let new_asset_id: AssetId = "tulip##ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland".parse().unwrap();
```

Comment:
Some tests just have "multihash@domain" literals, being independent of `test_samples`